packages/sharing-smart-contract/test/subscription.test.js

```diff
@@ -172,25 +239,12 @@ describe('Subscription.sol', () => {
 
   describe('setProtectedDataToSubscription()', () => {
```

Comment:
add test case
- cannot setProtectedDataToSubscription if the protected data is available for sale